components/Section.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
```

Comment:
The name Section.js is also used in my sidebar implementation. Must make sure they don't conflict

Reply:
This shouldn't be an issue as the name can be changed when the component is imported. Names are scoped per module (i.e. per file).